tests/integration/pull_review_test.go

```diff
@@ -161,10 +161,18 @@ func TestPullView_CodeOwner(t *testing.T) {
 			assert.NoError(t, err)
 
 			session := loginUser(t, "user5")
-			testPullCreate(t, session, "user5", "test_codeowner", true, forkedRepo.DefaultBranch, "codeowner-basebranch-forked", "Test Pull Request2")
+
+			// create a pull request on the forked repository, code reviewers should not be mentioned
+			testPullCreateDirectly(t, session, "user5", "test_codeowner", forkedRepo.DefaultBranch, "", "", "codeowner-basebranch-forked", "Test Pull Request2")
```

Comment:
title can be `Test Pull Request Fork`, as we already have `Test Pull Request2` in L136
Also, use `forkedRepo.OwnerName` and `forkedRepo.Name` can be more readable.